python/pynessie/nessie_client.py

```diff
@@ -164,15 +164,15 @@ def assign_tag(self: "NessieClient", tag: str, to_ref: str, old_hash: Optional[s
         tag_json = ReferenceSchema().dumps(Tag(tag, self.get_reference(to_ref).hash_))
         assign_tag(self._base_url, tag, tag_json, old_hash, self._ssl_verify)
 
-    def merge(self: "NessieClient", branch: str, to_branch: str, old_hash: Optional[str] = None) -> None:
+    def merge(self: "NessieClient", onto_branch: str, from_branch: str, old_hash: Optional[str] = None) -> None:
```

Comment:
Can you adopt the order of "from" and "onto" to the same as in cli.py (or vice versa)?

Reply:
done